projects/github.com/ZZROTDesign/docker-clean/package.yml

```diff
@@ -0,0 +1,22 @@
+distributable:
+  url: https://github.com/ZZROTDesign/docker-clean/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: ZZROTDesign/docker-clean
+
+provides:
+  - bin/docker-clean
+
+dependencies:
+  gnu.org/bash: '*'
```

Comment:
@jhheider I think any shell would work

Reply:
depending on bash is pretty tried-and-true to prevent issues, if you _need_ to depend on a shell.